soc/arm/nxp_kinetis/k6x/Kconfig.defconfig.mk66f18

```diff
@@ -0,0 +1,112 @@
+# FSL SEGGER K66F platform configuration options
+
+# Copyright (c) 2020 DENX Software Engineering GmbH
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_MK66F18
+
+config SOC
+	default "mk66f18"
+
+config K6X_CORE_CLOCK_DIVIDER
+	int "Freescale K6X core clock divider"
+	default 1
+	help
+	  This option specifies the divide value for the K6X processor core clock
+	  from the system clock.
+
+config K6X_BUS_CLOCK_DIVIDER
+	int "Freescale K6X bus clock divider"
+	default 2
+	help
+	  This option specifies the divide value for the K6X bus clock from the
+	  system clock.
+
+config K6X_FLEXBUS_CLOCK_DIVIDER
+	int "Freescale K6X FlexBus clock divider"
+	default 2
+	help
+	  This option specifies the divide value for the K6X FlexBus clock from the
+	  system clock.
+
+config K6X_FLASH_CLOCK_DIVIDER
+	int "Freescale K6X flash clock divider"
+	default 5
+	help
+	  This option specifies the divide value for the K6X flash clock from the
+	  system clock.
```

Comment:
Don't introduce new config symbols in a defconfig file

Reply:
@MaureenHelm Thanks, resolved.